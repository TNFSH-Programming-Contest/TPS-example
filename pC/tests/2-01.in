810986514186.658769 748439804307.400317 935375453462.738578
