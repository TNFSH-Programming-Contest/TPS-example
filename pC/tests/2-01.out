648910109546.70410156
